include/linux/slab_def.h

```diff
@@ -114,6 +115,8 @@ static inline unsigned int obj_to_index(const struct kmem_cache *cache,
 static inline int objs_per_slab_page(const struct kmem_cache *cache,
 				     const struct page *page)
 {
+	if (is_kfence_address(page_address(page)))
```

Comment:
What if we just move this check to memcg_alloc_page_obj_cgroups(), which seems to be the only caller of objs_per_slab_page? This way we'll avoid adding a dependency on kfence.h

Reply:
If we do that we can't easily test this function in kfence_test. There may also be future callers and things will break.

I'm willing to accept the include for sake of saner code. If we get push-back, we can either change it, or argue that this file should include its dependencies anyway (I actually don't know how the thing still works, given it refers to things that it doesn't include nor mm/slab.h includes -- probably by accident?! It's a mess, let's not make it messier ;-)).